packages/x-data-grid/src/components/containers/GridRootStyles.ts

```diff
@@ -41,6 +41,15 @@ const columnHeaderStyles = {
   },
 };
 
+declare module '@mui/material' {
+  interface Mixins {
+    DataGrid: {
+      containerBackground?: string;
+      pinnedBackground?: string;
```

Comment:
Would adding more variables from the [gridStyles](https://github.com/mui/mui-x/blob/3d86411074f2fe291d42edfa4b930f9baa951f62/packages/x-data-grid/src/components/containers/GridRootStyles.ts#L177-L197) here be helpful for some other use-cases?